### ISSUE TITLE
`css.global` doesn't vendor-prefix rules

### ISSUE THREAD

**Issue by keyanzhang** [threepointone/glamor#145]
Late to the game, but using glamor feels greeeeat! Thanks for making this.

I noticed rules that are inserted via `css.global(...)` don't get vendor-prefixed. Is this intentional? In my case, I'm using `css.global(...)` to style the node that ReactDOM renders to:

```js
css.global('#container', {
  minHeight: '100%',
  display: 'flex',
});
```

IMO it makes perfect sense for `css.insert` to insert a raw CSS string as-is, but it would be great if `css.global` can auto-prefix rules such that I won't need to manually add vendor prefixes. :)

**Comment by threepointone** [comment]
Good point, I hadn't thought of it and it slipped my mind. Will do!

**Comment by keyanzhang** [comment]
Thanks @threepointone!

**Comment by code-punkt** [comment]
Also, nesting and other shenanigans that are possible with `css` are not applied.
Not sure how much work that would be - totally fine to me without.

**Comment by threepointone** [comment]
yeah, I probably won't bring nesting etc into this. just a helper for inserting global css.

**Comment by threepointone** [comment]
published in 2.20.13
